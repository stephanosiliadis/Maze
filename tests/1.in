8 11
6 3 4 1 -1
7 3 3
1 4 33
5 2 2
2 4 5
2 7 8
3 6 15
1 7 7
6 7 12
6 5 1
4 8 10
8 5 20
