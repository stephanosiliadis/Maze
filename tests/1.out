42
1
7
3 R
7
2
5
6 R
5
2
4
8
